lib/main.js

```diff
@@ -96,6 +97,7 @@ globals.setImmediate(() => {
 });
 
 module.exports = runner.chain;
+module.exports.CallSite = CallSite;
```

Comment:
There could be a `CallSite.js` which reexports `CallSite` from `lib/assert.js`. Manipulating `runner.chain` like you do here doesn't seem ideal.